rewrite-gradle/src/main/java/org/openrewrite/gradle/util/GradleWrapper.java

```diff
@@ -59,15 +59,10 @@ public static Validated validate(ExecutionContext ctx,
 
         //noinspection unchecked
         return new Validated.Both(
-                Validated.test("distributionType", "must be a valid distribution type", distributionTypeName, dt -> {
-                    try {
-                        DistributionType.valueOf(dt);
-                        return true;
-                    } catch (Throwable e) {
-                        return false;
-                    }
-                }),
-                Semver.validate(version, null)
+            Validated.test("distributionType", "must be a valid distribution type", distributionTypeName,
+                dt -> Arrays.stream(DistributionType.values())
+                    .anyMatch(type -> type.name().equalsIgnoreCase(dt))),
```

Comment:
Will probably need to also determine what an empty/null distribution type means. Both `AddGradleWrapper` and `UpdateGradleWrapper` recipes allow null and internally default to `bin` within their visitors.

https://github.com/openrewrite/rewrite/blob/a4e8970dac822565b57f484202946c3a0280ba2d/rewrite-gradle/src/main/java/org/openrewrite/gradle/UpdateGradleWrapper.java#L67-L73

Reply:
if `distribution` type is null, then it internally defaults to `bin` at the beginning of this method. 
```
String distributionTypeName = distribution != null ? distribution : DistributionType.Bin.name().toLowerCase();
``` 
I think this is what you mean here. please let me know if I missed anything, thanks

Reply:
You are correct! I missed that part.